cattle/plugins/docker/compute.py

```diff
@@ -86,7 +88,8 @@ def _add_resources(self, ping, pong):
             'kind': 'docker',
             'name': Config.hostname(),
             'physicalHostUuid': physical_host['uuid'],
-            'uuid': DockerConfig.docker_uuid()
+            'uuid': DockerConfig.docker_uuid(),
+            'physicalHostStats': self.host_info.collect_data()
```

Comment:
Did you and @ibuildthecloud agree that physicalHostStats should be part of the compute ping response and that that's where cattle should expect it?

I ask because it feels like the data you're collecting doesn't need to be connected directly to what is going on in docker.compute. But, I don't have an immediate answer on how to best break this out completely on its own.